chrome-extension/src/App.js

```diff
@@ -18,13 +18,29 @@ function App() {
         return {...state,isPassword:false,focus:true,time:action.timeValue}
       case 'greetings':
         return {...state,greetings:action.greet}
+      case 'show-modal':
+        return {...state,showModal:!state.showModal}
       default:
         break;
     }
   }
 
-  const initilaState={userName:"",userEmail:"",showName:true,isEmail:false,isPassword:false,time:null,focus:false,greetings:""}
+  const initilaState={userName:"",userEmail:"",showName:true,isEmail:false,isPassword:false,time:null,focus:false,greetings:"",showModal:false}
+
   const [state,dispatch]=useReducer(reducerFunc,initilaState)
+  const [todoInput,setTodoInput]=useState()
+  const [todoValues,setTodoValues]=useState([])
+  const [editId,setEditId]=useState()
+
+  const addHandler=(value)=>{
+    setTodoValues([...todoValues,{id:uuid(),todoName:value,isDone:false}])
+  setTodoInput("")
+}
+
+  const todoInputHandler=(id)=>{
+  let newValue=todoValues.map(item=>item.id===id?{...item,isDone:!item.isDone}:item)
```

Comment:
try to use good variable name instead of `newValue` like newTodo

Reply:
ok